template/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="${require('./src/favicon.ico')}">
```

Comment:
Are you sure? `require("./src/favicon.js")` seems less magical and also consistent with other webpack-isms used (`require("./index.css")`).